allocation.md

```diff
@@ -157,6 +157,37 @@ Here is an example allocation for the `cost-model` container in a pod in Kubecos
 - `__unallocated__` refers to aggregated allocations without the selected `aggregate` field; e.g. aggregating by `label:app` might produce an `__unallocated__` allocation composed of allocations without the `app` label.
 - `__unmounted__` (or "Unmounted PVs") refers to the resources used by PersistentVolumes that aren't mounted to a Pod using a PVC, and thus cannot be allocated to a Pod.
 
+#### Negative Idle
+
+It is possible to have a negative idle cost that is technically valid in the following
```

Comment:
Is it worth saying you should confirm with Diagnostics that you're collecting data correctly?

Reply:
Added a note, what do you think?

Reply:
LGTM, but maybe add a link to Diagnostics docs if you have one that you like?

Reply:
Good idea, done.